udev-device.c

```diff
@@ -230,6 +230,23 @@ udev_device_get_sysattr_value(struct udev_device *ud, const char *sysattr)
 	return (NULL);
 }
 
+LIBUDEV_EXPORT int
+udev_device_set_sysattr_value(struct udev_device *ud, const char *sysattr, const char *value)
+{
+	char const *key;
```

Comment:
Why is this one `char const` while the parameter is `const char`? Seems  inconsistent.

The declaration might also be able to live inside the loop (L240)

Reply:
This was copied from `udev_device_get_sysattr_value()`. There are many other such inconsistenties, so I'd rather not fix this one yet.

Reply:
To be honest, I'm slighly unsure of the difference between char const *ptr and const char *ptr. There might be a reason to change this all over though, but it's outside scoop for this change.

Reply:
According to https://stackoverflow.com/a/890560/637669 there is no difference between `const char *` and `char const *`. It's only about style.